Tests/CommonTestTrait.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/*
+ * This file is part of the Sonata project.
+ *
+ * (c) 2018 - Sonata
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SonataHelpers\Tests;
+
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Trait CommonTestTrait
+ *
+ * @author Sylvain Rascar <sylvain.rascar@ekino.com>
+ */
+trait CommonTestTrait
```

Comment:
As this is not directly related to sonata, should we put this in this repository or in another one?